webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx

```diff
@@ -77,36 +92,51 @@ const WalletTxns: React.FunctionComponent<WalletTxnsProps> = (
   const [currentPage, setCurrentPage] = useState(1);
   const [tableRows, setTableRows] = useState<any[]>([]);
   const [includeRewards, setIncludeRewards] = useState(false);
-  const pageSize = WALLET_TXN_PAGE_SIZE;
-  const total = data ? data.length : 0;
+  const [pageSize, setPageSize] = useState(10);
+  const [pageSizeForFetch, setPageSizeForFetch] = useState(5);
+  const total = data || walletTxnCount ? data.length + walletTxnCount : 0;
   const pagesCount = Math.ceil(total / pageSize);
-  const to = (currentPage - 1) * pageSize + 1;
-  const from = Math.min(total, currentPage * pageSize);
-  const textLimit = 26;
```

Comment:
some of my changes are removed here

Reply:
@thedoublejay can you elaborate, post your alternative code section here or create another PR

Reply:
@monstrobishi I synced with @saurabh391 and we resolved it on his branch. Should be ok now.

Reply:
thanks!